db/Pos/DB/Class.hs

```diff
@@ -39,6 +39,10 @@ module Pos.DB.Class
        , DBIteratorClass (..)
```

Comment:
You forget to update the description of this module.

Reply:
What, do you think, should be added/removed? I will add needed documentation about getting raw data in CSL-1869, so documenting this part is not a strict requirement.

Reply:
> What, do you think, should be added/removed?

This: https://i.imgur.com/HYITtDr.png

Perhaps those changes were done before you wrote your question, but the thing is they were done after I wrote my comment.